### PR TITLE
Making CSS responsive with smaller devices and adding scrollbars

### PR DESCRIPTION
**Pull request by jreys**
Pull Request for Issue #100 
### Testing Instructions

Resize window and check if the behaviour is intended

### CODE REVIEW

File: media/com_associations/css/sidebyside.css [hunk 1]
```diff
@@ -9,9 +9,6 @@
 }
 
 .sidebyside #left-panel .inner-panel {
-	border-style: solid;
-	border-color: #999999;
-	border-width: 0 1px 0 0;
 	padding: 0 10px 0 0;
```
Comment (andrepereiradasilva) [inline media/com_associations/css/sidebyside.css]: if you remove the border css styles here you should remove the border in `html[dir=rtl] .sidebyside #left-panel .inner-panel` too...

**Comment by infograf768** [comment]
Works here.

**Comment by andrepereiradasilva** [comment]
doesn't work here. i get an horizontal scroll bar

image

**Comment by infograf768** [comment]
@andrepereiradasilva 
On which browser? Here on FF Mac, only the vertical bar

**Comment by infograf768** [comment]
@andrepereiradasilva 
For me the horizontal scroolbar happens ONLY when some stuff is larger than the width.
It is the case when I display for example a menu item of type single article where we have the Select Article field very large (the EDit button goes over the edge)
As soon as I display for the same menu item the Options tab, the horizontal scroll bar disapears.
 screen shot 2016-08-17 at 20 40 01

So I think this PR is OK

**Pull request merged**